app/models/user.rb

```diff
@@ -2,9 +2,14 @@ class User < ApplicationRecord
   has_many :user_groups
   has_many :groups, through: :user_groups
   has_many :lists
-  has_secure_password
+  has_many :received, foreign_key: :receiver_id, class_name: 'Invitation'
+  has_many :sent, foreign_key: :sender_id, class_name: 'Invitation'
   validates_uniqueness_of :email
+  validates_uniqueness_of :id, scope: [:user_groups, :groups]
```

Comment:
check if this is needed given line 4 of `UserGroup` model validations

Reply:
Removed.